test/LowerColorado_TX/test_AnA.yaml

```diff
@@ -57,6 +59,7 @@ compute_parameters:
         streamflow_da:
             #----------
             streamflow_nudging       : True
+            diffusive_streamflow_DA  : False

```

Comment:
maybe change the name of this parameter to `diffusive_streamflow_nudging` to be more in-line with naming convention.

Reply:
Got it.